src/hooks/use-global-data.js

```diff
@@ -0,0 +1,27 @@
+// @flow strict
+import { useStaticQuery, graphql } from 'gatsby';
+
+const useGlobalData = () => {
+  const data = useStaticQuery(
+    graphql`
+      query global {
+        allFile(filter: {relativePath: {eq: "logo.png"}}) {
+          edges {
+            node {
+              childImageSharp {
+                gatsbyImageData(width: 64, height: 64)
+              }
+              name
+            }
+          }
+        }
+      }
+    `
+  );
+
+  const logo = data.allFile?.edges[0]?.node ? data.allFile?.edges[0]?.node : undefined;
```

Comment:
Maybe throw an error or a warning. It seems like `logo` would be undefined by default so what's this check for?

Reply:
if logo not found, fields allFile and node are not present in the graphql response, and it make the site crash

Reply:
Ok thanks